Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/API/DriverOnboarding.yaml

```
@@ -186,6 +186,9 @@ types:
   PullDocumentReq:
     docType: DocumentType
     drivingLicenseNumber: Text
+  VerifyBankAccReq:
+      - bankAccountNo : Text
```

YAMLlint (1.38.0)

[warning] too many spaces before colon

(colons)